app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -82,13 +91,51 @@ class Overview extends React.Component {
     } else if (
       !nextProps.mappingWizardVisible &&
       !nextProps.planWizardVisible &&
+      hasMadeInitialPlansFetch &&
       !this.pollingInterval
     ) {
       fetchTransformationPlansAction(fetchTransformationPlansUrl);
       this.startPolling();
     }
   }
 
+  componentDidUpdate(prevProps) {
+    const { finishedTransformationPlans, addNotificationAction } = this.props;
+    const { hasMadeInitialPlansFetch } = this.state;
+
+    if (
+      hasMadeInitialPlansFetch &&
+      finishedTransformationPlans.length >
+        prevProps.finishedTransformationPlans.length
+    ) {
+      const oldMigrationIds = prevProps.finishedTransformationPlans.map(
+        plan => plan.id
+      );
+      const freshMigrations = finishedTransformationPlans.filter(
+        migration => !oldMigrationIds.includes(migration.id)
+      );
+
+      freshMigrations.forEach(plan => {
+        const planStatus =
+          plan.miq_requests[0].status.toLowerCase() === 'complete';
```

Comment:
oops, missed this my first time through.  A plan can have more than one request, for example, if a plan's first request fails, and then the user retries, the plan will have a total of 2 requests

We need to do something like
```javascript
const [mostRecentRequest] = plan.miq_requests.slice(-1)
```